modules/monitoring-creating-scrape-sample-alerts.adoc

```diff
@@ -71,3 +71,10 @@ spec:
 ----
 $ oc apply -f monitoring-stack-alerts.yaml
 ----
+
+. Additionally, you can check if a target has hit the configured limit:
+
+.. In the *Administrator* perspective of the web console, go to *Observe* -> *Targets* and select an endpoint with a `Down` status that you want to check.
+.. The *Scrape failed: sample limit exceeded* message is displayed if the endpoint failed because of an exceeded sample limit:
++
+image:monitoring-sample-limit-exceeded.png[Endpoint with exceeded sample limit]
```

Comment:
Not sure if the screenshot is necessary, considering it's the only one (AFAICT) in the assembly and isn't providing any new information really. Also the screenshot here is from a dark mode web console, whereas docs.openshift and the Customer Portal don't have a built-in dark mode to match, so it's a little jarring.